packages/ingest/src/S3ProviderClient.ts

```diff
@@ -145,6 +145,34 @@ class S3ProviderClient implements ProviderClient {
     }
   }
 
+  /**
+   * Upload a file
+   *
+   * @param {object} params
+   * @param {string} params.localPath - the full local file path
+   * @param {string} params.uploadPath - the full remote file path for uploading file to
+   * @param {string} params.remoteAltBucket - alternate per-file bucket override to this.bucket
```

Comment:
```suggestion
   * @param {string} [params.remoteAltBucket] - alternate per-file bucket override to this.bucket
```
https://jsdoc.app/tags-param.html